packages/api-v4/src/kubernetes/types.ts

```diff
@@ -28,6 +28,10 @@ export interface KubernetesCluster {
   id: number;
   k8s_version: string;
   label: string;
+  /**
+   * Upcoming Feature, Beta - Datastream logs, name of the property to be confirmed
+   */
+  logGeneration?: boolean;
```

Comment:
Using camel case in the API is incommon. Usually snake case is used.  

```suggestion
  log_generation?: boolean;
```

Is this a mistake or intentional?

Reply:
It was a temporary name. I've changed it now that we do have the final one.